### PR TITLE
Cleanup dependencies

### PR DESCRIPTION
**Pull request by joaomoreno**

### CODE REVIEW

**Review by roblourens — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -90,6 +90,7 @@
     "native-watchdog": "^1.4.1",
     "node-pty": "1.1.0-beta5",
     "tas-client-umd": "0.1.8",
+    "util": "^0.12.4",
```
Comment (connor4312) [inline package.json:93]: We can probably kill util too, it's used in exactly on place in the browser unit tests to promisify `tree-kill`

File: package.json [hunk 2]
```diff
@@ -195,11 +192,9 @@
     "mocha-multi-reporters": "^1.5.1",
     "npm-run-all": "^4.1.5",
     "opn": "^6.0.0",
-    "optimist": "0.3.5",
     "p-all": "^1.0.0",
```
Comment (connor4312) [inline package.json:195]: p-all is also used in exactly one place which could easily be replaced with a queue :)

**Review by connor4312 — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -178,7 +176,6 @@
     "gulp-untar": "^0.0.7",
     "husky": "^0.13.1",
     "innosetup": "6.0.5",
-    "is": "^3.1.0",
     "istanbul-lib-coverage": "^3.2.0",
     "istanbul-lib-instrument": "^5.2.0",
     "istanbul-lib-report": "^3.0.0",
```
Comment (connor4312) [inline package.json:181]: It's out of the range, but lazy.js is similarly easily replacable

File: resources/darwin/Credits.rtf [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1348\cocoasubrtf170
```
Comment (rzhao271) [inline resources/darwin/Credits.rtf:1]: Intentional?

Comment (roblourens) [reply resources/darwin/Credits.rtf:1]: I think that's just what rtf files look like under the hood

File: package.json (hunk 1 above)
Comment (joaomoreno) [reply package.json:93]: I believe it's used for webpack to compile extensions for browser targets. @aeschli please confirm.

File: resources/darwin/Credits.rtf (hunk 4 above)
Comment (joaomoreno) [reply resources/darwin/Credits.rtf:1]: No, not intentional! Thanks!

**Comment by joaomoreno** [comment]
@connor4312 Let's merge this in and then remove more stuff, please help me out 😄

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**